### DISCUSSION TITLE
Copilot -  Cannot proceed with payment

### DISCUSSION THREAD

**Discussion by dvantage** [community/community#51569] [Copilot Conversations]
### Select Topic Area

Question

### Body

When I click on the link https://github.com/github-copilot/signup, I see a message:

We are still investigating the issue with your GitHub account, and we will update you as soon as possible. Thank you for your patience during this process. We are sorry for any inconvenience

![image](https://user-images.githubusercontent.com/5517858/229116309-40825a49-b05a-4932-b212-9f20177dfd2a.png)

What are the problems with my PRO account?

**Comment by Akash1134** [comment]
Hi @dvantage Please contact our Support team for issues related to billing. Please [open a ticket](https://support.github.com/contact/account?tags=dotcom-footer) with them.

**Reply by dvantage** [reply]
> Hi @dvantage Please contact our Support team for issues related to billing. Please [open a ticket](https://support.github.com/contact/account?tags=dotcom-footer) with them.

Ticket created and no response for 3 days
